website/. browserslistrc

```diff
@@ -0,0 +1,7 @@
+# Browsers that we support
```

Comment:
Does the file name have an erroneous space between the `.` and the rest of the name?

Reply:
upppss